main.go

```diff
@@ -326,6 +351,16 @@ func main() {
 
 	rootCmd.AddCommand(deleteCmd)
 
+	rootCmd.AddCommand(editCmd)
+	editCmd.Flags().StringVarP(&editor, "editor", "", "",
+		strings.Trim(`
+The command to open a new editor. If this flag is not specified, it defaults to
+reading env vars "VISUAL", or "EDITOR" (in that order). If neither environment variable
+is found, these commands are attempted: "vi", "emacs", "nano", "pico". The command is
```

Comment:
I'm not sure we should do this. If `EDITOR` isn't set, I think we throw an error and say "set EDITOR"

Reply:
Sounds fine to me. I took the fallback to vi from git and added emacs so as to not be opinionated 😄